benchmark/hetero/at/fusion.vector.cpp.erb

```diff
@@ -2,24 +2,22 @@
 // Distributed under the Boost Software License, Version 1.0.
 // (See accompanying file LICENSE.md or copy at http://boost.org/LICENSE_1_0.txt)
 
-<% if n > 10 %>
-    #define FUSION_MAX_VECTOR_SIZE <%= ((n + 9) / 10) * 10 %>
```

Comment:
How come is this not required anymore? After all, we go until 100 elements for `fusion::vector`, and that should blow up the default limit, no?